src/app/cli/src/cli_commands.rs

```diff
@@ -542,3 +564,19 @@ where
 
     Ok(result_refs)
 }
+
+// ToDo add validation for DatasetRefPattern and DatasetRef expresion
+fn validate_many_dataset_wildcards<I>(
+    catalog: &dill::Catalog,
+    dataset_ref_patterns: I,
+) -> Result<Vec<DatasetRefPattern>, CLIError>
+where
+    I: IntoIterator<Item = DatasetRefPattern>,
+{
+    let mut result_refs = Vec::new();
```

Comment:
The result is no longer a ref, name of the collection is a bit confusing. Maybe `result_patterns`.